config.w32

```diff
@@ -224,6 +246,34 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no") {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
```

Comment:
Per my earlier comment, this is where I'd suggest defining values for all of the `MONGOCRYPT_` vars.